core/camera/src/main/java/com/google/jetpackcamera/core/camera/CameraSessionSettings.kt

```diff
@@ -58,6 +58,7 @@ internal sealed interface PerpetualSessionSettings {
  * The use cases typically will not need to be re-bound.
  */
 internal data class TransientSessionSettings(
+    val cameraInfo: CameraInfo,
```

Comment:
As discussed offline, the `TransientSessionSettings` are supposed to include settings that will not require the camera to be re-bound. You should find some other way to communicate the camera that should be bound outside of using `TransientSessionSettings`.

Reply:
sure. I saw "typically" and thought it would still be ok. I can move the cameraselector into a flow in `CameraXCameraUseCase`